tags/resources/tag-formation-templates/template-leadership-election-process.md

```diff
@@ -1,3 +1,15 @@
 # Leadership election process within the `<TAG NAME>`
```

Comment:
Perhaps we should have some direct link to https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#elections

Reply:
Yes 💯, this document will be added in a followup PR (see PR description). So I will resolve this for now, ok.